src/main.rs

```diff
@@ -3,7 +3,7 @@ extern crate time;
 use http::Request;
 use k8s_openapi::api::core::v1::Node;
 use kube::{api::ListParams, Api, ResourceExt};
-use log::{debug, trace};
+use log::{info, trace};
```

Comment:
Nit: You can also add this at the top and then you don't need to import each log level macro manually.

```rust
#[macro_use]
extern crate log;
```